Makefile

```diff
@@ -22,7 +22,7 @@ images: ansible/roles
 setup-infrastructure: ansible/roles check-deploy-env
 	cd ansible && ansible-playbook -e 'ansible_python_interpreter="/usr/bin/env python"' \
 		--tags "$(DEPLOY_ENV)" environments.yml
-	cd terraform && terraform init && terraform apply --auto-approve
+	cd terraform && terraform init && terraform apply -var epoch_version=0.16.0 --auto-approve
```

Comment:
so far so good, let's make this Makefile variable with default to 0.16.0:

- `EPOCH_VERSION =? 0.16.0` at makefile top
- `-var epoch_version=$(EPOCH_VERSION)` on this and following lines

Reply:
fixed